{"name": "Test login attempt with wrong credentials", "status": "passed", "steps": [{"name": "test_login_incorrect", "status": "passed", "parameters": [{"name": "login_data", "value": "('alex_art', 'test')"}, {"name": "driver", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"905defaa49d4df4b044cc37aca949bf8\")>"}], "start": 1537449168945, "stop": 1537449187546}], "parameters": [{"name": "login_data", "value": "('alex_art', 'test')"}], "start": 1537449168945, "stop": 1537449187546, "uuid": "7624fabf-277c-44a5-8ace-288dd3f9b8e7", "historyId": "eb21a96018a81d1b7c1af44bc9928f0c", "fullName": "src.tests.test_selenium.TestLogin#test_login_incorrect[login_data0]", "labels": [{"name": "story", "value": "Login"}, {"name": "tag", "value": "UI"}, {"name": "feature", "value": "User management"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_selenium"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "odp1210013"}, {"name": "thread", "value": "21332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_selenium"}]}